react-ui/src/pages/contest/Contest.jsx

```diff
@@ -222,6 +235,20 @@ function Contest() {
           >
             {name}
           </Typography>
+
+          <Accordion
+            className={classes.prompt}
+            expanded={isPromptOpen}
+            onChange={togglePromptOpen}
+          >
+            <AccordionSummary expandIcon={<ExpandMore />}>
+              Click here to view the prompt that contestants were given
```

Comment:
Could this be something like:
`${isPromptOpen ? 'Hide' : 'View'} contest prompt`

Feels like the sentence is a bit verbose.